CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 [1]: https://pypi.org/project/google-cloud-pubsub/#history
 
+### [1.4.3](https://www.github.com/googleapis/python-pubsub/compare/v1.4.2...v1.4.3) (2020-04-16)
+
+
+### Bug Fixes
+
+* fix docs warnings in Sphinx 3.0+ ([#70](https://www.github.com/googleapis/python-pubsub/issues/70)) ([21e761e](https://www.github.com/googleapis/python-pubsub/commit/21e761ee89a4c03e105dc9cddbab0a34be9a9fda))
```

Comment:
This is internal/chore, not really a bugfix?

Reply:
It comes from the PR description so I'd rather not change it at this point. Yes it's not 100% perfect. :)

Reply:
I'm fine with that, it's a minor thing anyway.